### ISSUE TITLE
What is the implementation in the lines?

### ISSUE THREAD

**Issue by TicTacToePP** [scott-yjyang/Vivim#12]
[            xz_s = xz.chunk(self.nframes, dim=-1)
            xz_s = torch.stack(xz_s,dim=-1)
            xz_s = xz_s.flatten(-2)](https://github.com/scott-yjyang/Vivim/blob/b089ad531bba451447b66a09430589d770e068fb/mamba/mamba_ssm/modules/mamba_simple.py#L245-L247)
            
I tested these lines and found xz_s is the same as original xz. Could you please help explain the lines? thanks!
